npm/test/index.test.ts

```diff
@@ -91,3 +91,28 @@ test("renderModules does not call authorisation when the front end info is missi
 
   keycloakInstance.mockRestore()
 })
+
```

Comment:
Probably need some more extensive unit tests. See suggestion about, about abstracting the functionality into separate functions to make unit testing easier.

Reply:
Tests finally working